bdbstore/src/test/java/org/apache/qpid/server/store/berkeleydb/replication/ReplicatedEnvironmentFacadeTest.java

```diff
@@ -1196,6 +1206,75 @@ public void onNoMajority()
                    masterListener.awaitForStateChange(State.MASTER, _timeout, TimeUnit.SECONDS));
     }
 
+    @Test
+    public void testNodeCommitNoSync()  throws Exception
```

Comment:
please move code to populate and  read test db into a separate method. Something like below

private void addTestKeyValue(Database db, Transaction txn,  int key, String value)
private String getTestKeyValue(Database db,  int key)

That should improve test readability..

Reply:
Done